src/os/bluestore/BlueStore.cc

```diff
@@ -988,6 +973,21 @@ void BlueStore::LRUCache::_trim(uint64_t onode_max, uint64_t buffer_max)
   }
 }
 
+void BlueStore::LRUCache::_trim_buffers_to(uint64_t max) {
```

Comment:
nit: open brace goes on next line in .cc files

Reply:
fixed!